### DISCUSSION TITLE
Mute function with external speaker

### DISCUSSION THREAD

**Discussion by olgkcn** [MonitorControl/MonitorControl#1124] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I am using MonitorControl 4.0.2  on my Macbook Pro M1 with a Dell S2722DC monitor via USB-C cable. There is no problem with ordinary usage but when I connect a speaker to a monitor I can't mute the volume.

### Expected behavior

The mute function should work when an external speaker connected to the monitor.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.3.1
- Mac model: MacBook Pro (13-inch, M1, 2020)
- MonitorControl version: 4.0.2
- Monitor(s): Dell S2722DC
- Monitor Cable(s)/Connection(s): Type C
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by nskaiser** [comment]
I have a very similar issue w/ the same setup.  I also am plugging in external speaker to the Monitor audio out. I have disabled the speakers in the Dell config.  The Mac volume controls work, but at the same time, they re-enable the Dell speaker output which I don't want. I want the Dell internal speakers to remain off always.

**Comment by waydabber** [comment]
This is entirely dependent on the display firmware implementation of the relevant DDC command. There is a theoretical DDC setting to specify the audio input but it is not implemented by any of the display manufacturers afaik. I am afraid there is no much to do with this on MonitorControl side.

**Comment by georgealways** [comment]
Chiming in to say I've had an issue with mute using a very similar setup. My Macbook is connected to my monitor via USB-C, and the monitor to speakers via 3.5mm. When I mute, I can still hear audio, just incredibly faintly.

```
- macOS version: 12.5
- Mac model: MacBook Pro (16-inch, M1, 2020)
- MonitorControl version: 4.1.0
- Monitor(s): LG 34WK95U
- Monitor Cable(s)/Connection(s): Type C
- Apple Silicon/M1 (yes or no): Yes
```

Volume controls work perfectly otherwise. Thanks for the great app!

**Comment by waydabber** [comment]
You can try using the alternate mute method, that helps in some cases. These kinds of issues are display firmware specific.
